src/main/resources/ERCode/ERCodeEditSheet.xml

```diff
@@ -7,12 +7,12 @@
   <defaultLanguage/>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1407942174000</creationDate>
   <parent>ERCode.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <creationDate>1407942174000</creationDate>
-  <date>1408009358000</date>
-  <contentUpdateDate>1408009358000</contentUpdateDate>
+  <date>1436344693000</date>
+  <contentUpdateDate>1436344693000</contentUpdateDate>
```

Comment:
It's a good practice to skip these changes when committing modifications on XWiki documents, since they are not actual code changes that need to be versioned or that need to end up in the end-user's XWiki instance.

Just something to consider for the future.